pages/api/archive.js

```diff
@@ -0,0 +1,20 @@
+import { dbConnect } from '../../src/lib/db/database';
+import Archive from '../../src/lib/models/Archive';
+import Task from '../../src/lib/models/Task';
+
+export default async function handler(req, res) {
+	if (req.method === 'POST') {
+		const oldTaskData = JSON.parse(req.body);
+		await dbConnect();
+		await Archive.insertMany(oldTaskData);
+		const deleteMany = await Task.deleteMany({ done: true });
+
+		console.log(req.body);
+		res.status(200).json({
```

Comment:
A successful post request should return a status code 201 (CREATED)

Reply:
True & done.
thx